chiselFrontend/src/main/scala/Chisel/When.scala

```diff
@@ -2,9 +2,12 @@
 
 package Chisel
```

Comment:
Which follows the very simple and standardized convention of what transforms do, and isn't that much code at all...